### DISCUSSION TITLE
[FEATURE] Github Gist for mobile

### DISCUSSION THREAD

**Discussion by MrHakan** [community/community#5614] [Mobile] [answered]
Please add Github Gists for mobile version of github.

**Comment by amrehman** [comment, marked as answer]
Thanks a lot for your feedback!

I've added your note to our internal issue where we are tracking adding gists. If you have any other feedback or feature requests, please keep them coming!

**Comment by AlexHedley** [comment]
#5332
#5921
#6466

**Comment by Popaj13** [comment]
> #5332
> #5921
> #6466

**Comment by kanition** [comment]
Still nothing in 2025
